### PR TITLE
Move to use Webpack

### PR DESCRIPTION
**Pull request by nwhite89**
Starts to build rb-table using Webpack #16

### CODE REVIEW

File: src/directives/action-button/action-button.tpl.html [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<a ng-repeat="button in config.buttons"
+    class="{{ button.btnClass }}"
```
Comment (chrisvxd) [inline src/directives/action-button/action-button.tpl.html]: indentation

File: src/directives/button/index.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+     * @description
+     * Renders a button with the columns title text as the button text.
+     * Emits event when button clicked.
+     *
+     * @ngInject
+     */
+    function rbTableButtonCellRenderer () {
+        return {
+            restrict: 'E',
+            // @ngInject
```
Comment (chrisvxd) [inline src/directives/button/index.js]: Is this ngInject needed?

File: src/directives/cell/index.js [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+    /**
+     * @ngdoc directive
+     * @name rb-table:rbTableCell
+     * @restrict A
+     * @element td
+     * @scope
+     *
+     * @description
+     * Used internally by rbTable.
```
Comment (chrisvxd) [inline src/directives/cell/index.js]: This description sucks

File: src/directives/html/index.js [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+     * @example
+     * columns = {
+     * 		'data': {
+     * 			directive: 'rb-table-html-cell-renderer',
+     * 			config: {
+     * 				html: '<div>{{ itemValue }}</div>'
+     * 			}
+     * 		}
+     * }
+     * // Will output <div>Some cell data</div>
```
Comment (chrisvxd) [inline src/directives/html/index.js]: inline comment?

File: src/directives/table/index.js [hunk 5]
```diff
@@ -0,0 +1,57 @@
+define([
+    '../../controllers/table',
```
Comment (chrisvxd) [inline src/directives/table/index.js]: Can this get aliased? I don't think we should ever be doing relative directories

File: test/unit/src/directives/html/html.spec.js [hunk 6]
```diff
@@ -0,0 +1,87 @@
...
+                {
+                    'id': 3,
+                    'name': 'Terry Bradley',
+                    'contact_details': {
+                        'email': 'terry.bradley@example.com'
+                    },
+                    'date_joined': '2014-06-01T13:00:00Z',
+                    'age': 54,
+                    'number': 5
+                }
```
Comment (chrisvxd) [inline test/unit/src/directives/html/html.spec.js]: Are the tests going to use the same mock data as in your next PR?

File: test/unit/src/directives/ui-state-link/ui-state-link.spec.js [hunk 7]
```diff
@@ -0,0 +1,199 @@
...
+
+        beforeEach(inject(function ($rootScope, $compile, $q) {
+            scope = $rootScope.$new();
+            scope.data = mockData;
+            scope.columns = columnsMock;
+            q = $q;
+        }));
+
+        function compileDirective () {
+            inject(function ($compile) {
```
Comment (jonman) [inline test/unit/src/directives/ui-state-link/ui-state-link.spec.js]: I don't think you need to inject `$compile` here as it is already available in above `beforeEach`

**Pull request merged**